src/output-json-alert.c

```diff
@@ -243,6 +243,10 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
         }
     }
 
+    if (p->flow) {
+        json_object_set_new(js, "app_proto",
```

Comment:
IMO, 2. may be a better idea, getting metric information could be interesting. Maybe we could do: print partial flow record or output app_proto ?